src/main/java/codeu/model/data/User.java

```diff
@@ -31,12 +32,14 @@
    * @param name the username of this User
    * @param passwordHash the password hash of this User
    * @param creation the creation time of this User
+   * @param phoneNumber the phoneNumber of this User
    */
-  public User(UUID id, String name, String passwordHash, Instant creation) {
+  public User(UUID id, String name, String passwordHash, Instant creation, String phoneNumber) {
```

Comment:
It's not really necessary to change the constructor since you're passing null to it pretty much everywhere. A setter method for the few times you have a phone number you want to supply would be fine (and that would allow reverting a bunch of the otherwise unchanged files)